docs/clients/iquart_flight_controller_interface.rst

```diff
@@ -119,14 +120,14 @@ Message Table
 
 Type ID 88 | IQUART Flight Controller Interface
 
-+--------+--------------+----------------+-----------+------+-------------------------------------------------------------------------------------------------------------------+
-| Sub ID | Short Name   | Access         | Data Type | Unit | Note                                                                                                              |
-+========+==============+================+===========+======+===================================================================================================================+
-| 1      | telemetry    | get            | uint8     |      | Returns 16 bytes of telemetry information. See the :ref:`IFCITelemetryData struct<ifcitelemetrydata_note>` notes. |
-+--------+--------------+----------------+-----------+------+-------------------------------------------------------------------------------------------------------------------+
-| 2      | throttle_cvi | get, set, save | uint8     |      | The control value index for the throttle.                                                                         |
-+--------+--------------+----------------+-----------+------+-------------------------------------------------------------------------------------------------------------------+
-| 3      | x_cvi        | get, set, save | uint8     |      | The control value index value for the x rectangular coordinate.                                                   |
-+--------+--------------+----------------+-----------+------+-------------------------------------------------------------------------------------------------------------------+
-| 4      | y_cvi        | get, set, save | uint8     |      | The control value index value for the y rectangular coordinate.                                                   |
-+--------+--------------+----------------+-----------+------+-------------------------------------------------------------------------------------------------------------------+
++--------+--------------+----------------+-----------+-----------------+-------------------------------------------------------------------------------------------------------------------+
+| Sub ID | Short Name   | Access         | Data Type | Unit            | Note                                                                                                              |
++========+==============+================+===========+=================+===================================================================================================================+
+| 1      | telemetry    | get            | uint8     | 16 Byte Array   | Returns 16 bytes of telemetry information. See the :ref:`IFCITelemetryData struct<ifcitelemetrydata_note>` notes. |
```

Comment:
The data type here isn't uint8. I'd say the data type is the 16 Byte Array really and the unit is tough, its really many different units all packed together. Maybe the unit should just be a link to the Telemetry data struct like you have in the note?